ChangeLog

```diff
@@ -1,4 +1,5 @@
 Version 2016.05.11:
+	* Use HTTPS instead of Git protocol when cloning. (davidjb)
```

Comment:
Just a note:  changelog usually updated according to the commit log right before next bootstrap stable version will be released.